hermes/probe/probe.go

```diff
@@ -149,22 +140,24 @@ func (mp *Probe) runProbe(ctx context.Context, metricChan chan<- *cpmetrics.Even
 	var wg sync.WaitGroup
 	for _, t := range mp.targets {
 		wg.Add(1)
-		go func(t *Target) {
+		go func(t *target.Target) {
 			defer wg.Done()
-			var err error
-			if t.LatencyMetrics, err = metrics.NewMetrics(mp.config, t.Target); err != nil {
-				mp.logger.Errorf(err.Error())
-				return
+			if t.LatencyMetrics == nil {
+				var err error
+				if t.LatencyMetrics, err = metrics.NewMetrics(mp.config, t.Target); err != nil {
+					mp.logger.Errorf(err.Error())
+					return
+				}
 			}
+
 			probeCtx, _ := context.WithDeadline(ctx, time.Now().Add(mp.interval()))
 			start := time.Now()
 			exitStatus, err := mp.runProbeForTarget(probeCtx, t)
```

Comment:
Nit: Change the name of the `exitStatus` variable, as it currently suggests that `runProbeForTarget` executes a subprocess.

Reply:
Renamed to `status`.